lib/pbench/server/api/resources/users_api.py

```diff
@@ -238,10 +238,14 @@ def post(self):
             self.logger.exception("Exception while logging in a user")
             abort(HTTPStatus.INTERNAL_SERVER_ERROR, message="INTERNAL ERROR")
 
+        self.logger.info(
+            "auth_token: ({}!r): ({})", auth_token, auth_token.decode("UTF-8")
+        )
         response_object = {
-            "auth_token": auth_token,
+            "auth_token": auth_token.decode("UTF-8"),
```

Comment:
Oops; this was a mistake, attempting to workaround a problem caused by a bad PyJwt on my test VM.

Reply:
They have WiFi in the Pemigewasset Wilderness??  😈

Reply:
Not necessarily _fast_ WiFi, but, yeah.